.github/ISSUE_TEMPLATE/android-issue.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Android Issue
+about: Template for logging issues on Android
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- Have you searched for similar issues? Before submitting this issue, please check the open issues and add a note before logging a new issue. 
+
+PLEASE USE THE TEMPLATE BELOW TO PROVIDE INFORMATION ABOUT THE ISSUE. 
+INSUFFICIENT INFO WILL GET THE ISSUE CLOSED. IT WILL ONLY BE REOPENED AFTER SUFFICIENT INFO IS PROVIDED-->
+
+## Description <!-- Provide a brief description of the issue -->
+
+
+## Steps to reproduce <!-- Please add a series of steps to reproduce the issue -->
+
+   1. 
+   2. 
+   3. 
+
+## Actual result <!-- Please add screenshots if needed -->
+
+
+## Expected result
+
+
+## Issue reproduces how often <!-- [Easily reproduced/Intermittent issue/No steps to reproduce] -->
+
+
+## Issue happens on <!-- Mention yes or no -->
+   - Current Play Store version?
+   - Beta build?
+
+## Device details
+   - Install type (ARM, x86):
+   - Device (Phone, Tablet, Phablet):
+   - Android version:
+   
+## Brave version
+
+
+### Website problems only
```

Comment:
We should probably add the following in this section as well:

```
- Does the issue resolve itself when disabling Brave Rewards?
```

Reply:
Updated